client/app/app.js

```diff
@@ -0,0 +1,10 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+var WikiPoetryApp = require('./components/WikiPoetryApp.react')
+
+
+// Render FluxCartApp Controller View
+ReactDOM.render(
+  <WikiPoetryApp />,
+  document.getElementById('flux-wiki')
```

Comment:
Instead of having this getElementByID, I think we can just have like 8 say "<WikiPoetryApp, flux-wiki>". It's something about ids being put in the global scope when they are defined.